packages/backend/src/file-import/file-import.controller.ts

```
@@ -0,0 +1,116 @@
+    @Param('workspaceId') workspaceId: string,
+    @UploadedFile() file: Express.Multer.File,
+  ): Promise<{ id: string }> {
+    if (!file) {
+      throw new BadRequestException('No file uploaded');
+    }
+    // `'data'` is what keeps the declared type out of the decision: the filter
+    // above already vouched for the name, so a `Content-Type` of `image/png`
+    // on an `x.csv` cannot move this upload into the image category — where it
+    // would meet a 25 MB cap after being buffered against a 200 MB one, and be
+    // stored at the bucket root instead of the expiring `imports/` prefix.
+    return this.fileService.upload(file.buffer, file.mimetype, file.originalname, {
```

ESLint

[error] Replace `file.buffer,·file.mimetype,·file.originalname,` with `⏎······file.buffer,⏎······file.mimetype,⏎······file.originalname,⏎·····`

(prettier/prettier)